docs/general/ddl/data-types.rst

```diff
@@ -2867,6 +2862,45 @@ constant array values.
 
 .. _data-types-geo:
 
+Nested arrays
+..............
+
+Nested arrays cannot be used directly in column definitions  (i.e. 
+``ARRAY(ARRAY(DOUBLE))`` is not accepted), but multiple arrays can be nested
+as long as objects come between them (e.g. ``ARRAY(OBJECT AS (innerarray 
+ARRAY(DOUBLE)))``).
+
+Nested arrays can still be used directly in input and output to UDFs:
+
+::
+
+    CREATE OR REPLACE FUNCTION sort_nested_array(  "data" ARRAY(ARRAY(DOUBLE)),
```

Comment:
```suggestion
    CREATE OR REPLACE FUNCTION sort_nested_array("data" ARRAY(ARRAY(DOUBLE)),
```

Reply:
Here and everywhere in code blocks, please remove unnecessary whitespaces and check identation.